util/chpl-completion.bash

```diff
@@ -0,0 +1,72 @@
+# file: chpl-completion
+# chpl parameter completion
+
+# Tab autocompletion for chpl options and directories/.chpl files in bash
+# Source this file to enable this autocompletion.  To automatically enable
+# autocompletion, source this file from your .bashrc file.
+
+# This file was generated with the script:
+# $CHPL_HOME/util/devel/gen-chpl-bash-completion
+
+# This is modeled after the example at:
+# http://tldp.org/LDP/abs/html/tabexpansion.html
+# and Dennis's answer to this question:
+# https://superuser.com/questions/564716/bash-completion-for-filename-patterns-or-directories
+
+_chpl ()
+{
+  local cur
+  COMPREPLY=()
+  cur=${COMP_WORDS[COMP_CWORD]}
+  case "$cur" in
+    -*)
+      # The following lists were generated with the commands preceding them.
```

Comment:
Maybe remove these comments and just direct people to util/devel/gen-chpl-bash-completion? (I doubt we'll remember to update these comments whenver that script gets updated)

Reply:
Sounds good.  I'll just remove the comments since there's already a comment at the top of the script pointing at `gen-chpl-bash-completion`.